scripts/mkuidefaults.py

```diff
@@ -39,7 +40,7 @@ def chunks(l, n):
 QCoreApplication.setApplicationName("QGIS3")
 
 if len(sys.argv) == 1:
-    print "Usage: python ./scripts/mkuidefaults.py \"location_to_ini\""
+    print("Usage: python ./scripts/mkuidefaults.py \"location_to_ini\"")
```

Comment:
Recommending `python ...` here and using `python3` as shebang on top looks inconsistent to me. Maybe better switch it to `Usage: ./scripts/mkuidefaults.py \"path_to_ini\"` would be better with the shebang?